services/user/Config/plugin/src/lib.rs

```diff
@@ -0,0 +1,65 @@
+#[allow(warnings)]
+mod bindings;
+use bindings::*;
+
+use exports::config::plugin::branding::Guest as Branding;
+use exports::config::plugin::producers::Guest as Producers;
+
+use host::common::types::Error;
+
+use crate::bindings::exports::config::plugin::producers::Producer;
+use staged_tx::plugin::proposer::set_propose_latch;
+
+use psibase::services::producers::SERVICE as PRODUCERS_SERVICE;
+
+struct ProposeLatch;
+
+impl ProposeLatch {
+    fn new(app: psibase::AccountNumber) -> Self {
+        set_propose_latch(Some(&app.to_string())).unwrap();
```

Comment:
Technically I think this raii isn't really needed anymore since the propose latch can now only be set by the top-level app.